From 4e7a0bee0cc91d85a17bcc35a65da661799da90f Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:42:25 -0400
Subject: [alsa] ctxfi: ctpcm.c printk replacement

Message-id: <140972823573-439-patchset-jkysela@redhat.com>
Patchwork-id: 91138
O-Subject: [RHEL 7.1 PATCH 438/442] ALSA: ctxfi: ctpcm.c: printk replacement
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

replaced printk with corresponding pr_err

Signed-off-by: Sudip Mukherjee <sudip@vectorindia.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>

(picked from kernel: 57f2d8b797c4c8d9e65e3b9fae98246be5a93df3)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/ctxfi/ctpcm.c b/sound/pci/ctxfi/ctpcm.c
index e8a4feb..6826c2c 100644
--- a/sound/pci/ctxfi/ctpcm.c
+++ b/sound/pci/ctxfi/ctpcm.c
@@ -217,7 +217,7 @@ static int ct_pcm_playback_prepare(struct snd_pcm_substream *substream)
   err = atc->pcm_playback_prepare(atc, apcm);
 
  if (err < 0) {
-  printk(KERN_ERR "ctxfi: Preparing pcm playback failed!!!\n");
+  pr_err("ctxfi: Preparing pcm playback failed!!!\n");
   return err;
  }
 
@@ -324,7 +324,7 @@ static int ct_pcm_capture_prepare(struct snd_pcm_substream *substream)
 
  err = atc->pcm_capture_prepare(atc, apcm);
  if (err < 0) {
-  printk(KERN_ERR "ctxfi: Preparing pcm capture failed!!!\n");
+  pr_err("ctxfi: Preparing pcm capture failed!!!\n");
   return err;
  }
 
@@ -435,7 +435,7 @@ int ct_alsa_pcm_create(struct ct_atc *atc,
  err = snd_pcm_new(atc->card, "ctxfi", device,
      playback_count, capture_count, &pcm);
  if (err < 0) {
-  printk(KERN_ERR "ctxfi: snd_pcm_new failed!! Err=%d\n", err);
+  pr_err("ctxfi: snd_pcm_new failed!! Err=%d\n", err);
   return err;
  }
 
-- 
1.7.1